kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKms.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kms.aws;
+
+import java.net.URI;
+
+import io.kroxylicious.systemtests.executor.Exec;
+import io.kroxylicious.systemtests.executor.ExecResult;
+
+/**
+ * The type Aws kms local.
```

Comment:
In what sense is this "local".   Should this say 'cloud?'

Reply:
Yeah, copy paste issue. Thanks!